src/analysis/retail/paladin/protection/modules/Abilities.js

```diff
@@ -148,48 +147,48 @@ class Abilities extends CoreAbilities {
         },
       },
       {
-        spell: SPELLS.DIVINE_STEED.id,
+        spell: TALENTS.DIVINE_STEED_TALENT.id,
         category: SPELL_CATEGORY.UTILITY,
         cooldown: 45,
-        charges: combatant.hasTalent(SPELLS.CAVALIER_TALENT) ? 2 : 1,
+        charges: combatant.hasTalent(TALENTS.CAVALIER_TALENT) ? 2 : 1,
         gcd: {
           base: 1500,
         },
       },
       {
-        spell: SPELLS.BLESSING_OF_FREEDOM.id,
+        spell: TALENTS.BLESSING_OF_FREEDOM_TALENT.id,
         category: SPELL_CATEGORY.UTILITY,
         cooldown: 25,
         gcd: {
           base: 1500,
         },
       },
       {
-        spell: SPELLS.BLESSING_OF_PROTECTION.id,
+        spell: TALENTS.BLESSING_OF_PROTECTION_TALENT.id,
         isDefensive: true,
         category: SPELL_CATEGORY.UTILITY,
         cooldown: 5 * 60,
         gcd: {
           base: 1500,
         },
-        enabled: !combatant.hasTalent(SPELLS.BLESSING_OF_SPELLWARDING_TALENT),
+        enabled: !combatant.hasTalent(TALENTS.BLESSING_OF_SPELLWARDING_TALENT),
```

Comment:
This should probably be `combatant.hasTalent(TALENTS.BLESSING_OF_PROTECTION_TALENT)` since the two aren't exclusive anymore, as I understand it.
```suggestion
        enabled: combatant.hasTalent(TALENTS.BLESSING_OF_PROTECTION_TALENT),
```

Reply:
swapped to check for blessing of protection now that they are no longer mutually exclusive